addons/website/tests/test_iap.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import odoo
+import odoo.tests
+from odoo.addons.iap.tools import iap_tools
+
+@odoo.tests.tagged('external')
+class TestIap(odoo.tests.HttpCase):
+
+    def test_01_industries_lang(self):
+        """Ensure that the industries are translated in all the languages
+        supported by the configurator."""
+        with iap_tools.enabled_iap_test():
+            configurator_languages = ['en', 'de', 'es', 'fr', 'hr', 'hu', 'id', 'it', 'mk', 'nl', 'pt', 'ru', 'zh']
```

Comment:
Note that I will add `ar` when [this other PR](https://github.com/odoo/iap-apps/pull/641) will be merged :+1: